Dfe.Academies.External.Web/Models/ConversionApplicationContributor.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Dfe.Academies.External.Web.Models;
+
+public class ConversionApplicationContributor
+{
+    public ConversionApplicationContributor(string name, string role)
+    {
+        Name = name;
+        Role = role;
+    }
+
+    public long Id { get; set; }
+    public string Name { get; set; }
+
+    public string Role { get; set; }
```

Comment:
Do we have an enum for Role? I know there's only Chair in the roles enum so far but I think we might need to challenge David on that.

Reply:
yeah we do, i didn't realize it was the same until I was looking at another page this afternoon